docs/guide/personalization.md

```diff
@@ -426,14 +448,16 @@ This file is responsible for sending notifications to recommendation API after t
 To render recommended content, use a dedicated `showRecommendationsAction` from the `RecommendationController.php`:
 
 ``` html+twig
-render_esi(controller(‘EzRecommendationClientBundle:Recommendation:showRecommendations’, {
-               ‘contextItems’: content.id,
-               ‘scenario’: ‘front’,
-               ‘outputTypeId’: ‘blog_post’,
-               ‘limit’: 3,
-               ‘template’: ‘EzRecommendationClientBundle::recommendations.html.twig’,
-               ‘attributes’: [‘title’, ‘intro’, ‘image’, ‘uri’]
-           }))
+{% if ez_recommendation_enabled() and not location.isDraft %}#}
```

Comment:
This condition always should be use when we have custom templates with some html blocks. In case if recommendation credentials are not provided then we will be have empty html blocks. This is why we use this conditions in demo but you missed <div class="container"></div> inside condition. By default using this condition is not necessary. 

Remove second condition because someone can change "location" parameter and it break template rendering.